### PR TITLE
Set OOMScoreAdjust to -999 for containerd

### PR DESCRIPTION
**Pull request by raviprpandey**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Without the above change, containerd has a oom_score_adj value of 0. This makes containerd vulnerable to OOM kills if node comes under pressure. With the above change, containerd is in the last set of candidates to be OOM killed.

###### Change Log  

- Set oom_score_adj value for containerd to -999

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

**Review by anphel31 — COMMENTED** [review]
You’ll need to bump the release number of this spec as well

**Comment by raviprpandey** [comment]
> You’ll need to bump the release number of this spec as well

you mean this version number? https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/moby-containerd/moby-containerd.spec#L7 .
What should this be bumped to? Its currently set to 1.6.22

**Comment by anphel31** [comment]
> > You’ll need to bump the release number of this spec as well
> 
> you mean this version number? https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/moby-containerd/moby-containerd.spec#L7 . What should this be bumped to? Its currently set to 1.6.22

The release number should be bumped, in this case to 5. The release is listed on this line: https://github.com/microsoft/CBL-Mariner/blob/main/SPECS/moby-containerd/moby-containerd.spec#L8C19-L8C19

**Comment by raviprpandey** [comment]
> > > You’ll need to bump the release number of this spec as well
> > 
> > 
> > you mean this version number? https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/moby-containerd/moby-containerd.spec#L7 . What should this be bumped to? Its currently set to 1.6.22
> 
> The release number should be bumped, in this case to 5. The release is listed on this line: https://github.com/microsoft/CBL-Mariner/blob/main/SPECS/moby-containerd/moby-containerd.spec#L8C19-L8C19

Thanks for the reference. Fixed.

**Comment by anphel31** [comment]
You will also want to add an entry to the changelog for this new release number

**Comment by anphel31** [comment]
Please also update the "moby-containerd.signatures.json" file with the new hash, and run a buddy build

**Comment by raviprpandey** [comment]
> Please also update the "moby-containerd.signatures.json" file with the new hash, and run a buddy build

Ran budddy build: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=471701&view=results

File: SPECS/moby-containerd/moby-containerd.spec [hunk 1]
```diff
@@ -90,6 +90,9 @@ fi
 %dir /opt/containerd/lib
 
 %changelog
+* Tue Dec 20 2023 Ravi Prakash Pandey <rapandey@microsoft.com> - 1.6.22-2
```
Comment (anphel31) [inline SPECS/moby-containerd/moby-containerd.spec:93]: The version number here should be `1.6.26-2`

**Review by anphel31 — APPROVED** [review]
Approving, as this looks good to me from a packaging point of view. As to the containerd service change, I'll defer to the CBL-Mariner-Container-Runtime group.

File: SPECS/moby-containerd/moby-containerd.spec [hunk 2]
```diff
@@ -90,6 +90,9 @@ fi
 %dir /opt/containerd/lib
 
 %changelog
+* Tue Dec 20 2023 Ravi Prakash Pandey <rapandey@microsoft.com> - 1.6.26-2
```
Comment (ms-mahuber) [inline SPECS/moby-containerd/moby-containerd.spec:93]: Wed, not Tue

**Comment by ms-mahuber** [comment]
I am end-to-end testing this commit for moby-containerd-cc: https://github.com/microsoft/CBL-Mariner/commit/5169e3f1b68ff40fe6a4140f8a1f6bd5245bc3c2

**Comment by ms-mahuber** [comment]
> I am end-to-end testing this commit for moby-containerd-cc: [5169e3f](https://github.com/microsoft/CBL-Mariner/commit/5169e3f1b68ff40fe6a4140f8a1f6bd5245bc3c2)

Tests in internal pipelines looking good.

**Review by ms-mahuber — APPROVED** [review]

**Review by anphel31 — APPROVED** [review]

**Pull request merged**